packages/optimizer/lib/transformers/SeparateKeyframes.js

```diff
@@ -45,9 +42,28 @@ class SeparateKeyframes {
   constructor(config) {
     this.log_ = config.log.tag('SeparateKeyframes');
     this.minify = config.minify !== false;
+    this.enabled = !!config.separateKeyframes;
+    if (
+      this.enabled &&
+      (!isDependencyInstalled('postcss') ||
```

Comment:
purely from a DX standpoint, would it make sense to have `isDependencyInstalled` take an array?

```js
const areDependenciesInstalled = require('../isDependencyInstalled');


areDependenciesInstalled(['postcss','postcss-safe-parser', 'cssnano-simple'])
```